### ISSUE TITLE
Support to http urls

### ISSUE THREAD

**Issue by Athanax** [justsoft/video_thumbnail#10]
the package returns errors when the video url is http. i.e. not secure

**Comment by justsoft** [comment]
Give this a try? Let me the result thank you!
https://stackoverflow.com/questions/51902629/how-to-allow-all-network-connection-types-http-and-https-in-android-9-pie

**Comment by Athanax** [comment]
> Give this a try? Let me the result thank you!
> https://stackoverflow.com/questions/51902629/how-to-allow-all-network-connection-types-http-and-https-in-android-9-pie

This has helped so great. 
**AndroidManifest.xml**

    android:usesCleartextTraffic="true" 

I had three cases of URLs.

1. Localhost: http://127.0.0.1:8000 which I had set to http://10.0.2.2:8000 for use on emulators.

2. HTTP for not secure.

3. HTTPS for secure.

The solution only worked for cases 2 and 3 which I think to me am okay because I've hosted the API.

About issue #9, the reason was I tested the application using HTTP and https as in cases 2 and 3 above on a mobile device and in localhost (case 1) on an Android TV device which I think for now it's solved.
